tribute.html

```diff
@@ -15,7 +15,6 @@ <h1 id="title">Cat familly</h1>
         <figure id="img-div">
             <img id="image" src="https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&w=800">
             

```

Comment:
- [ ] I kindly suggest that you always add the reason "why", because even if you know and it seems obvious to you, the person you are reviewing might not know thus causing more confusion to them 😉